cider/include/cider/CiderRuntimeModule.h

```diff
@@ -150,6 +151,7 @@ class CiderRuntimeModule {
   constexpr static size_t kMaxOutputRows = 1000;
 
   bool is_group_by_;
+  bool is_using_arrow_fromat_;
```

Comment:
can not get from compileOption?

Reply:
operator doesn't save compileOption,
compileOption is a private member of Runtimemudule,
so will add a private member in operator, and set it when initialization